pkgs/applications/audio/curseradio/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, python3Packages, mpv }:
+
+python3Packages.buildPythonApplication rec {
+  version = "0.2";
+  name    = "curseradio-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "chronitis";
+    repo   = "curseradio";
```

```suggestion
    repo   = pname;
```

Usually reuse `pname` there also.